### PR TITLE
Add option for Blogger Importer

### PR DESCRIPTION
**Pull request by pablinos**
This is a new PR based on #24286 by @talldan but including a suggested change to a related PR #24350. The suggestion is to include any string translations in the `render` method, so that they are updated as necessary.

To test the changes follow the instructions in #24286

### CODE REVIEW

File: client/my-sites/importer/importer-header.jsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ class ImporterHeader extends React.PureComponent {
 	};
 
 	getLogo = icon => {
-		if ( includes( [ 'wordpress', 'medium' ], icon ) ) {
+		if ( includes( [ 'wordpress', 'medium', 'blogger-alt' ], icon ) ) {
```
Comment (jblz) [inline client/my-sites/importer/importer-header.jsx]: Just curious... Why `blogger-alt` & not just `blogger`?

Comment (pablinos) [reply client/my-sites/importer/importer-header.jsx]: Good question! It's using `social-logos` to display the icon and looking at the Blogger one it includes the border, which we don't want:

https://github.com/Automattic/social-logos/blob/master/svg-min-react/blogger.svg

The alt version is just the B logo which is better for styling.

**Comment by pablinos** [comment]
It's strange about the upload problem. @bisko Was having similar problems with the upload being cancelled. Is that happening for all import uploads or just this one?

**Comment by bisko** [comment]
The upload problem happens with Strict Site Isolation enabled in Chrome. The interesting part is that if you toggle it off and on it will start working again, without issues, until it fails again. I'm trying to figure out a reproducible test case, so I can report a Chrome bug.

File: client/my-sites/site-settings/section-import.jsx [hunk 2]
```diff
@@ -146,21 +149,28 @@ class SiteSettingsImport extends Component {
 							</header>
 						</CompactCard>
 
-						{ ( ! hasMediumActiveImports || hasWpActiveImports ) &&
+						{ ( ( ! hasMediumActiveImports && ! hasBloggerActiveImports ) || hasWpActiveImports ) &&
```
Comment (dmsnell) [inline client/my-sites/site-settings/section-import.jsx]: hrm. these are looking a bit silly. should we be consolidating these into a single selector?

Comment (pablinos) [reply client/my-sites/site-settings/section-import.jsx]: Yes this was spotted as it was getting worked on, but the decision was made to break the changes up in order to for the PRs to each have a single purpose.

@talldan addressed this problem in #24349 which I'm looking to update and merge.

**Pull request merged**